packages/theme/css/characteristics/text.css

```diff
@@ -149,10 +157,6 @@
 
   --uitk-text-help-fontSize: 11px;
   --uitk-text-help-lineHeight: 14px;
-
-  --uitk-text-fontSize: 12px;
-  --uitk-text-lineHeight: 16px;
-  --uitk-text-minHeight: 16px;
```

Comment:
min height has been deleted?

Reply:
this is used in pill, checkbox, card, grid...

Reply:
wouldn't 'line-height' be enough? why do we need min-height on text?

Reply:
added back in https://github.com/jpmorganchase/uitk/pull/260